mlflow/tracking/default_experiment/registry.py

```diff
@@ -47,7 +47,7 @@ def register_entrypoints(self):
             except (AttributeError, ImportError) as exc:
                 warnings.warn(
                     "Failure attempting to register default experiment"
-                    + 'context provider "{}": {}'.format(entrypoint.name, str(exc)),
+                    + f'context provider "{entrypoint.name}": {exc}',
```

Comment:
is `+` redundant?

Reply:
Yeah, I'll remove this in a follow-up.

Reply:
@gabrielfu Ruff has https://beta.ruff.rs/docs/rules/explicit-string-concatenation. Do you want to enable this?